lib/commonConfig.js

```diff
@@ -131,6 +138,29 @@ function processEnvironmentVariables() {
         config.providerUrl = process.env.IOTA_PROVIDER_URL;
     }
 
+    if (process.env.IOTA_AUTH_ENABLED) {
```

Comment:
CNR entry missing?

Reply:
fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/705/commits/d5b833acab98d0cd47bb4fc96b512d7a3aa65846